README.md

```diff
@@ -21,8 +21,12 @@ under mycroft.conf
     "ssl": false,
     // in mycroft-core all skills share a bus, this allows malicious skills
     // to manipulate it and affect other skills, this option ensures each skill
-    // gets it's own websocket connection
-    "shared_connection": true
+    // gets its own websocket connection
+    "shared_connection": true,
+    // filter out messages of certain types
```

Comment:
nitpick, clarify this just filters it from logs , otherwise it seems they arent emitted to other connected clients